src/llama.cpp

```diff
@@ -5052,6 +5055,7 @@ static void llm_load_hparams(
                 ml.get_key(LLM_KV_SSM_INNER_SIZE,     hparams.ssm_d_inner);
                 ml.get_key(LLM_KV_SSM_STATE_SIZE,     hparams.ssm_d_state);
                 ml.get_key(LLM_KV_SSM_TIME_STEP_RANK, hparams.ssm_dt_rank);
+                ml.get_key(LLM_KV_SSM_B_DT_RMS, hparams.ssm_b_dt_rms);
```

Comment:
```suggestion
                ml.get_key(LLM_KV_SSM_B_DT_RMS, hparams.ssm_b_dt_rms, false);
```

This is not a required key.

It's better not to make previous Mamba conversions fail to load.

Reply:
Perfect, thanks for handling backward compatibility!